CMakeLists.txt

```diff
@@ -43,6 +43,12 @@ IF(ZLIB_FOUND)
 	LIST(APPEND AXL_LINK_LINE "-lz")
 ENDIF(ZLIB_FOUND)
 
+# PTHREADS
+FIND_PACKAGE(Threads REQUIRED)
+IF(THREADS_HAVE_PTHREAD_ARG)
+	LIST(APPEND AXL_LINK_LINE "-lpthread")
```

Comment:
Use `AXL_EXTERNAL_LIBS` instead. The cmake’s findThreads module is documented here: https://cmake.org/cmake/help/v3.14/module/FindThreads.html which lists what variables it is setting. I think we want to use the `_INIT` variable??